react/ProductSummaryDescription.tsx

```diff
@@ -11,9 +11,10 @@ const CSS_HANDLES = ['description'] as const
 
 interface Props {
   classes?: CssHandlesTypes.CustomClasses<typeof CSS_HANDLES>
+  maxSizeDescription?: number
```

Comment:
Lgtm, I just wanted to suggest renaming this to `descriptionMaxSize` or `maxDescriptionSize`. Otherwise it seems like this is for setting the description of the max size (at least that's how I understand it when reading).